library/common/jni/jni_interface.cc

```diff
@@ -800,6 +800,12 @@ Java_io_envoyproxy_envoymobile_engine_EnvoyHTTPFilterCallbacksImpl_callReleaseCa
 
 // EnvoyHTTPStream
 
+extern "C" JNIEXPORT jint JNICALL Java_io_envoyproxy_envoymobile_engine_JniLibrary_readData(
```

Comment:
I can't remember if we have to add these functions to proguard?

Reply:
I think just new clases on the JVM side.